app/src/main/java/org/literacyapp/authentication/StudentImageCollectionActivity.java

```diff
@@ -245,6 +277,14 @@ public void run() {
                     studentImageDao.insert(studentImage);
                 }
                 Log.i(getClass().getName(), "storeStudentImages has finished successfully.");
+
+                // Initiate background job ad-hoc for face recognition training
+                ComponentName componentNameFaceRecognitionTranining = new ComponentName(getApplicationContext(), FaceRecognitionTrainingJobService.class);
+                JobInfo.Builder builderFaceRecognitionTranining = new JobInfo.Builder(0, componentNameFaceRecognitionTranining);
```

Comment:
I'm not really sure how the job id works. But the value should probably be different from another Job, am I right? If that's the case, make sure 0 is not already used as id for other jobs.